README.md

```diff
@@ -124,7 +124,7 @@ npm run lint:js:fix
 
 ## Deployment
 
-Please refer this [link](https://consoledot.pages.redhat.com/docs/dev/developer-references/design/frontend.html)
+Please refer this [link](https://docs.google.com/document/d/1PKGLs1zaBvSyOGHQqjVEfADRyCiBknzmGLJjZvNlw7s/edit?usp=sharing) (Needs VPN)
```

Comment:
Do you mean we need to connect with the VPN to access the doc?

Reply:
@r14chandra not the doc but to access few links inside the doc.

Reply:
Okay, I think you can add this in the doc itself as it is a bit misleading.